CMakeLists.txt

```diff
@@ -7,7 +7,8 @@ set(VERSION_REGEX "version[\t ]*=[\t ]*([0-9]+\\.[0-9]+\\.*[0-9]*)([-]([a-z]+))*
 if (NOT (${VERSION_CONFIG} MATCHES ${VERSION_REGEX}))
   message(FATAL_ERROR "Cannot read version from version.config")
 endif ()
-
+# a hack to avoid change of SQL extschema variable
```

Comment:
Can you explain why this is necessary / what this does? I'm not sure I follow.

Reply:
PostgresQL uses @extschema@ as a pre-populated variable to refer to target schema. Coincidentally, @@ is also the syntax to use CMake variables from inside code files that are being string-replaced during cmake. I define such a CMake variable so that after this variable replacement sql script is technically not changed.